src/resmom/linux/mom_mach.c

```diff
@@ -8050,8 +8050,8 @@ pidcache_reset(pidcachetype_t *pidcache)
 	while (errno = 0, (dent = readdir(cpusetdir)) != NULL) {
 		if (!isdigit(dent->d_name[0]))
 			continue;
-
-		sprintf(taskname, "%s/%s/tasks", cpusetfs, dent->d_name);
+                /* cpusetfs is 18 chars , '//tasks' is 7 chars, so taking 230 chars from dent->d_name */
+		snprintf(taskname, sizeof(taskname), "%.18s/%.230s/tasks", cpusetfs, dent->d_name);
```

Comment:
This warning was fixed by using pbs_asprintf(), not using string formatting.

Reply:
Done